### PR TITLE
Feat/liquidations feed

### PR DESCRIPTION
**Pull request by p-meier**
**Before submitting this PR, please make sure**:

- [ x] Your code builds clean without any errors or warnings
- [ x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

The base for this code was layed in Office-Hours Session of Botcamp Cohort 7 together with @cardosofede. 

The idea here is to fetch the forceOrder websocket stream of binance to be able to react to liquidation events that happen in the market. It is architected in the same way as the candles-feed and it is easy to add streams from other exchanges. 

**Tests performed by the developer**:

- Unit-Tests which verify the functionality
- Running the example-strategy with and without trading pairs (= all liquidations) and checking with `status --live` the correctness of the functionality. Also that the older ones get deleted after the amount of seconds specified as `max_retention_seconds` have passed

**Tips for QA testing**:

There is a simple test strategy in `scripts/liquidations_example.py` that can be used to verify the behaviour.

### CODE REVIEW

File: test/hummingbot/data_feed/liquidations_feed/test_liquidations_factory.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import unittest
+
+from hummingbot.data_feed.liquidations_feed.binance import BinancePerpetualLiquidations
+from hummingbot.data_feed.liquidations_feed.liquidations_factory import LiquidationsConfig, LiquidationsFactory
+
+
+class TestCandlesFactory(unittest.TestCase):
```
Comment (cardosofede) [inline test/hummingbot/data_feed/liquidations_feed/test_liquidations_factory.py:7]: TestLiquidationsFactory*

**Review by cardosofede — CHANGES_REQUESTED** [review]
Great work! only change the name of the last test class

File: test/hummingbot/data_feed/liquidations_feed/test_liquidations_factory.py (hunk 1 above)
Comment (p-meier) [reply test/hummingbot/data_feed/liquidations_feed/test_liquidations_factory.py]: just done, this one I forgot as it seems

**Comment by p-meier** [comment]
@cardosofede I saw that the test-coverage is slightly below 80% and added a few more tests to satisfy the test coverage metric. 

Hope it now builds and is ready to be merged.

**Review by cardosofede — APPROVED** [review]

**Pull request merged**